checkov/terraform/checks/resource/gcp/GoogleComputeBootDiskEncryption.py

```diff
@@ -0,0 +1,21 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+
+class GoogleComputeBootDiskEncryption(BaseResourceCheck):
```

Comment:
@arielkru Can this be refactored into `BaseResourceValueCheck`?

Reply:
@nimrodkor The scan logic doesn't match the use case, it checks for 2 existing keys